conda_build/convert.py

```diff
@@ -32,7 +32,7 @@
 @echo off
 set PYFILE=%~f0
 set PYFILE=%PYFILE:~0,-4%-script.py
-"%~f0\\..\\..\\python.exe" "%PYFILE%" %*
+"%~dp0\..\python.exe" "%PYFILE%" %*
```

Comment:
Wow! Makes sense, I guess. It must just do the parsing of all the .. before anything else. Thanks for teaching me.